docs/data-operate/import/import-way/insert-into-values-manual.md

```diff
@@ -0,0 +1,305 @@
+
+insert_max_filter_ratio
+
+- Default value: 1.0
+
+- Description: Since version 2.1.5. Only effective when `enable_insert_strict` is false. Used to control the error tolerance when using `INSERT INTO VALUES`. The default value is 1.0, which means all errors are tolerated. It can be a decimal between 0 and 1. It means that when the number of error rows exceeds this ratio, the INSERT task will fail.
+
+### Return values
+
+INSERT INTO VALUES is a SQL statement, and it returns a JSON string in its results.
+
+Parameters in the JSON string:
+
+| Parameter | Description                                                  |
+| --------- | ------------------------------------------------------------ |
+| Label     | Label of the import job: can be specified using "INSERT INTO tbl WITH LABEL label..." |
+| Status    | Visibility of the imported data: If it is visible, it will be displayed as "visible." If not, it will be displayed as "committed." In the "committed" state, the import is completed, but the data may be delayed in becoming visible. There is no need to retry in this case.`visible`: The import is successful and the data is visible.`committed`: The import is completed, but the data may be delayed in becoming visible. There is no need to retry in this case.Label Already Exists: The specified label already exists and needs to be changed to a different one.Fail: The import fails. |
+| Err       | Error message                                                |
+| TxnId     | ID of the import transaction                                 |
+
+**Successful INSERT**
+
+```sql
+mysql> INSERT INTO test_table (user_id, name, age) VALUES (1, "Emily", 25), (2, "Benjamin", 35), (3, "Olivia", 28), (NULL, "Alexander", 60), (5, "Ava", 17);
+Query OK, 5 rows affected (0.05 sec)
+{'label':'label_26eebc33411f441c_b2b286730d495e2c', 'status':'VISIBLE', 'txnId':'61071'}
+```
+
+`Query OK` indicates successful execution. `5 rows affected` indicates that a total of 5 rows of data have been imported. 
+
+**Successful INSERT with warnings**
+
+```sql
+mysql> INSERT INTO test_table (user_id, name, age) VALUES (1, "Emily", 25), (2, "Benjamin", 35), (3, "Olivia", 28), (NULL, "Alexander", 60), (5, "Ava", 17);
+Query OK, 4 rows affected, 1 warning (0.04 sec)
+{'label':'label_a8d99ae931194d2b_93357aac59981a18', 'status':'VISIBLE', 'txnId':'61068'}
+```
+
+`Query OK` indicates successful execution. `4 rows affected` indicates that a total of 4 rows of data have been imported. `1 warnings` indicates the number of rows that were filtered out. 
+
+You can use the [SHOW LOAD](../../../sql-manual/sql-statements/Show-Statements/SHOW-LOAD/) statement to view the filtered rows.
```

Comment:
new path : ../../../sql-statements/data-modification/load-and-export/SHOW-LOAD.md